testData/solutions/win32.json5

```diff
@@ -6139,34 +6139,122 @@
+                            "title": "Enhance Inputs",
+                            "description": "Emphasize links, buttons, menus, textfields, and other inputs.",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    },
+                    "lineSpace": {
+                        "schema": {
+                            "title": "Line Space",
+                            "description": "Adjusts the spacing between lines of text.",
+                            "type": "number",
+                            "default": 1,
+                            "minimum": 1,
+                            "maximum": 3,
+                            "multipleOf": 0.1
+                        }
+                    },
+                    "selectionTheme": {
+                        "schema": {
+                            "title": "Selection Theme",
+                            "description": "Change the highlight colour for text selections.",
+                            "type": "string",
+                            "default": "default",
+                            "enum": [
+                                "default",
+                                "yellow",
+                                "green",
+                                "pink"
+                            ]
+                        }
+                    },
+                    "selfVoicingEnabled": {
+                        "schema": {
+                            "title": "Self Voicing",
+                            "description": "Let the device read site content aloud.",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    },
+                    "simplifiedUiEnabled": {
+                        "schema": {
+                            "title": "Simplified UI",
+                            "description": "Only display the main content.",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    },
+                    "syllabificationEnabled": {
+                        "schema": {
+                            "title": "Syllabification",
+                            "description": "Display words broken down into their syllables.",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    },
+                    "tableOfContentsEnabled": {
+                        "schema": {
+                            "title": "Table of Contents",
+                            "description": "Create a table of contents.",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
```

Comment:
Thank you for alphabetising these, it makes it much easier to keep track of things IMO.

Reply:
Yah, i should have done that from the beginning. It was getting pretty hard to find things.